server/project/Dependencies.scala

```diff
@@ -61,6 +64,46 @@ object Dependencies {
     val nak     = "org.scalanlp" %% "nak"            % "1.3" exclude("org.scalanlp", "breeze_2.11") exclude("org.scalanlp", "breeze-natives_2.11")
   }
 
+  object spark {
+    val version = "1.2.0"
+
+    val core = ("org.apache.spark" %% "spark-core" % version)
+      .exclude("org.slf4j", "slf4j-log4j12")
+      .exclude("org.eclipse.jetty.orbit", "javax.transaction")
+      .exclude("org.eclipse.jetty.orbit", "javax.mail")
+      .exclude("org.eclipse.jetty.orbit", "javax.mail.glassfish")
+      .exclude("org.eclipse.jetty.orbit", "javax.activation")
+      .exclude("commons-beanutils", "commons-beanutils-core")
+      .exclude("commons-collections", "commons-collections")
+      .exclude("commons-logging", "commons-logging")
+      .exclude("com.esotericsoftware.minlog", "minlog")
+      .exclude("org.slf4j", "slf4j-api")
+      .exclude("org.apache.hadoop", "hadoop-yarn-api")
+    val mllib = ("org.apache.spark" %% "spark-mllib" % version)
+      .exclude("org.slf4j", "slf4j-api")
+    val streaming = "org.apache.spark" %% "spark-streaming" % version
+    val streamingKafka = ("org.apache.spark" %% "spark-streaming-kafka" % version)
+      .exclude("commons-beanutils", "commons-beanutils")
+      .exclude("commons-collections", "commons-collections")
+      .exclude("com.esotericsoftware.minlog", "minlog")
+  }
+
+  object akkaAnalytics {
+    val version = "0.2"
+
+    val cassandra = ("com.github.krasserm" % "akka-analytics-cassandra_2.10" % version)
+      .exclude("com.typesafe.akka", "akka-actor_2.10")
+      .exclude("com.esotericsoftware.minlog", "minlog")
+      .exclude("commons-beanutils", "commons-beanutils-core")
+      .exclude("commons-collections", "commons-collections")
+      .exclude("org.slf4j", "jcl-over-slf4j")
+      .exclude("org.slf4j", "slf4j-api")
+      .exclude("org.apache.spark", "spark-core_2.10")
+      .exclude("commons-logging", "commons-logging")
+  }
```

Comment:
Is it better to `exclude` here or to use `provided`? Reference https://github.com/sbt/sbt-assembly#-provided-configuration for the question "trigger"!